### PR TITLE
Updating Supported files

### PR DESCRIPTION
**Pull request by mlonsk**
Creating documentation for supporting file types

### CODE REVIEW

File: te3/features/supported-files.md [hunk 1]
```diff
@@ -1,19 +1,108 @@
...
+
+### DAX files (.dax)
+
+DAX queries are expressions that can be used to manipulate and analyze data in tabular models. A DAX file is a text file that contains one or more DAX queries. You can save a DAX file in Tabular Editor 3 and use it later to run the queries again. You can also open a DAX file in other tools that support DAX, such as [DAX Studio](https://daxstudio.org).
+
+### C# Scripts (.csx)
+
+Creating and editing C# Scripts is one of Tabular Editor's biggest productivity features.
+
+These scripts can be saved as files with the .csc extension and loaded into Tabular Editor as well as saved as Macros.
```
[the remark this answers is not in the document]
Comment (mlonsk) [reply te3/features/supported-files.md:90]: I have been considering if this page should also include files in the %LocalApp% folder such as Macros, BPA rules etc. Your comment makes me think it should.

File: te3/features/supported-files.md [hunk 2]
```diff
@@ -1,19 +1,108 @@
...
+### Diagram file (.te3diag)
+
+A .te3diag file is a file format that stores the diagram of a model created with TE3. 
+
+These file can be useful for documenting the model structure and logic for other developers who work on the same project. A .te3diag file can be saved in the same folder as the model file for easy access and reference.
+
+
+### DAX files (.dax)
+
+DAX queries are expressions that can be used to manipulate and analyze data in tabular models. A DAX file is a text file that contains one or more DAX queries. You can save a DAX file in Tabular Editor 3 and use it later to run the queries again. You can also open a DAX file in other tools that support DAX, such as [DAX Studio](https://daxstudio.org).
```
[the remark this answers is not in the document]
Comment (mlonsk) [reply te3/features/supported-files.md:84]: Yes, I think we should.

File: te3/features/supported-files.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mlonsk) [reply te3/features/supported-files.md:144]: I have added documentation for the extra files

File: te3/features/supported-files.md [hunk 3]
```diff
@@ -1,19 +1,206 @@
...
+![Supported File Types BIM](~/images/file-types/te3-supported-file-bim.png)
+
+[Download example .bim file ](https://raw.githubusercontent.com/TabularEditor/TabularEditorDocs/main/assets/file-types/bim-file-example.bim)
+
+
+### [Power BI](#tab/PowerBI)
+
+Tabular Editor can handle two types of Power BI files the Power BI Template file (.pbit) and the Power BI Project file (.pbip).
+
+The main difference between the two, is that the **.pbip file contains model data**, where as the **.pbit file contains no data**. Both file types contain model **metadata** in the JSON-based [Tabular Object Model (TOM)](https://docs.microsoft.com/en-us/analysis-services/tom/introduction-to-the-tabular-object-model-tom-in-analysis-services-amo?view=asallproducts-allversions) format, which can be loaded by Tabular Editor.
```
[the remark this answers is not in the document]
Comment (mlonsk) [reply te3/features/supported-files.md]: Fixed in latest commit

**Review by otykier — APPROVED** [review]

**Pull request merged**